src/DependencyInjection/Container.php

```
@@ -62,6 +62,20 @@ public function get($name, $invalidBehavior = ContainerInterface::EXCEPTION_ON_I
   /**
    * {@inheritdoc}
    */
+  public function set($id, $service, $scope = self::SCOPE_CONTAINER) {
```

I think we should throw an exception instead, because this Container is frozen as soon as it has been "compiled".